Makefile.am

```diff
@@ -282,16 +293,6 @@ EXTRA_DIST += \
   vms/pcre2.h_patch \
   vms/stdint.h
 
-# These files are used in the preparation of a release
-
-EXTRA_DIST += \
-  PrepareRelease \
-  CheckMan \
-  CleanTxt \
-  Detrail \
-  132html \
-  doc/index.html.src
```

Comment:
I don't feel we need to be bundling into the tarball all our maintainer scripts.

Reply:
Indeed not, nowadays. I think they were originally added way back, certainly before GitHub, probably even before Svn, when tarball was the only way to download anything. Ah yes, its documented as happening in 2007.